File: API/API 12.5 oficial/UnimindAPI/Prova.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("Prova", Schema = "unimind")]
public class Prova
{
    [Key]
    [Column("idProva")]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
    public int IdProva { get; set; }

    [Column("idFonte")]
    public int IdFonte { get; set; }

    [Column("ano")]
    public int Ano { get; set; }

    [Column("qntdQuestoes")]
    public int QntdQuestoes { get; set; }

    [Column("fase")]
    public string Fase { get; set; }
}

File: app/src/main/java/com/example/unimind/data/Prova.kt
package com.example.unimind.data

import com.google.gson.annotations.SerializedName

data class Prova(
    @SerializedName("idProva") val idProva: Int,
    @SerializedName("idFonte") val idFonte: Int,
    @SerializedName("ano") val ano: Int,
    @SerializedName("qntdQuestoes") val qntdQuestoes: Int,
    @SerializedName("fase") val fase: String?
)